MH4U Dex/DamageZonesCollectionViewController.m

```diff
@@ -36,45 +31,50 @@ @implementation DamageZonesCollectionViewController
 - (void)viewDidLoad
 {
     [super viewDidLoad];
-    self.flowLayout.itemSize = CGSizeMake(CGRectGetWidth(self.collectionView.bounds), 40.0f);
+
     self.damageZones = [self.monster.damageZone sortedArrayUsingDescriptors:@[[NSSortDescriptor
                                                                                sortDescriptorWithKey:DamageZoneAttributes.id
                                                                                ascending:YES]]];
 }
 
+- (void)viewWillLayoutSubviews
+{
+    CGFloat cellSize = CGRectGetWidth(self.collectionView.bounds);
```

Comment:
I'd recommend adding a comment to almost all layout code to remind Future-You what you Present-You was thinking.